### ISSUE TITLE
Comments don't show on the "demo" page

### ISSUE THREAD

**Issue by Giszmo** [fiatjaf/nocomment#32]
https://fiatjaf.com/nostr.html is linked as a demo page, yet there is no comments and commenting doesn't work.

**Comment by damascene** [comment]
> it was removed because it was full of spam 
Fiatjaf - Dec. 2024

**Comment by fiatjaf** [comment]
Yeah, I have to make something to do relay-based filtering for comments to avoid spam -- soon.
